mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulContestsScreen.kt

```diff
@@ -84,13 +84,18 @@ fun StatefulTournamentScreen(
     currentUser: AuthenticatedUser,
     onTournamentClick: (TournamentReference) -> Unit,
     onNewContestClickAction: () -> Unit,
+    onFilterClick: () -> Unit,
```

Comment:
Missing documentation.

Reply:
Fixed in 9a134a69295616820a0cccaeb459af8d3fd831ed.